pkg/local_object_storage/writecache/writecache.go

```diff
@@ -3,8 +3,8 @@ package writecache
 import (
 	"sync"
 
-	"github.com/nspcc-dev/neofs-node/pkg/core/object"
 	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/fstree"
+	objectSDK "github.com/nspcc-dev/neofs-sdk-go/object"
```

Comment:
Can we just replace the import without an alias? There is no conflict.

Reply:
As far as I remember, @carpawell wanted to have the same import aliases for all SDK and V2 components? 
https://github.com/nspcc-dev/neofs-node/pull/1131#issuecomment-1031437700

Reply:
ok, if there are no conflicts, it could be done in such way